Upgrade branch 2.7.x using TemplateControl

octonato opened the pull request:
```
Updated with template-control on 2019-01-10T23:13:53.360Z
  /LICENSE:
    wrote /LICENSE
  /NOTICE:
    wrote /NOTICE
  /.travis.yml:
    wrote /.travis.yml
  **/plugins.sbt:
    addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.7.0-RC9")
  **build.gradle:
    def playVersion = "2.7.0-RC9"

```

renatocaval commented:
Needs a new webjars-play release 

Depends on https://github.com/webjars/webjars-play/pull/84

octonato committed:
Updated with template-control on 2019-01-14T16:37:13.407Z
  /LICENSE:
    wrote /LICENSE
  /NOTICE:
    wrote /NOTICE
  **build.sbt:
    libraryDependencies += "org.webjars" %% "webjars-play" % "2.7.0-RC9"
  **/plugins.sbt:
    addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.7.0-RC9")
  **build.gradle:
    def playVersion = "2.7.0-RC9"
  **build.gradle:
        play "org.webjars:webjars-play_\$scalaVersion:2.7.0-RC9

octonato committed:
Manual fix for build.gradle file
